documentation/source/documentation/cheatsheets/python.rst

```diff
@@ -0,0 +1,600 @@
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    proc(arg1, arg2, *args)       |    apply(proc, arg1, arg2, args)              |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    map(proc, list1, list2, ...)  |    map(proc, list1, list2, ...)               |
+|    # returns iterator in         |                                               |
+|    # Python 3                    |                                               |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    map(proc, vec1, vec2, ...)    |    map(proc, vec1, vec2, ...)                 |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    map(proc, str1, str2)         |    map(proc, str1, str2)                      |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    for a,b in zip(list1, list2): |    do(proc, list1, list2)                     |
+|      proc(a, b)                  |                                               |
++----------------------------------+-----------------------------------------------+
+
+Continuations
+-------------
+
+Python doesn’t have first-class continuations like Scheme’s
+``call/cc``, and Dylan’s ``block`` syntax is more like structured
+exception handling or generators.
+
+Here are approximate equivalents:
+
++-----------------------------------+-----------------------------------------------+
+| Python                            | Dylan                                         |
++===================================+===============================================+
+| .. code-block:: python            | .. code-block:: dylan                         |
+|                                   |                                               |
+|    def example():                 |    block (k)                                  |
+|      try:                         |      body                                     |
+|        body()                     |    exception (e :: <error>)                   |
+|      except Error:                |      handle-error()                           |
```

Comment:
`except Error as e:` is closer to the Dylan side.  (Or you can change the Dylan side to `exception (<error>)`.)